volume/volume.go

```diff
@@ -290,6 +297,14 @@ func ParseMountSpec(cfg mounttypes.Mount, options ...func(*validateOpts)) (*Moun
 				mp.Propagation = cfg.BindOptions.Propagation
 			}
 		}
+	case mounttypes.TypeTmpfs:
+		rawOptions, err := convertTmpfsOptions(cfg.TmpfsOptions)
```

Comment:
Please refer to https://github.com/docker/docker/pull/26837#discussion_r84215734